### DISCUSSION TITLE
Daisy UI

### DISCUSSION THREAD

**Discussion by 42willow** [catppuccin/catppuccin#2285] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

https://daisyui.com/theme-generator/

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by Liumingxun** [comment]
I create an issue for this port #2379, you can try it. @42Willow 
Welcome to provide feedback on it.

**Comment by sgoudham** [comment]
Closing as further discussion should take place in #2379
